### PR TITLE
feature(MAG-1704): add screenshot testing to F2S

### PR DESCRIPTION
**Pull request by ParthJadhav**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,17 +9,34 @@ const tmpDir = require('./tmp-dir');
 
 const inputFileName = core.getInput('file-name');
 const awsOutputDir = core.getInput('aws-output-directory');
-const outputFileName = inputFileName.replace(/\.xd$/, `.${awsOutputDir}`);
+const conversionType = core.getInput('conversion-type');
+let outputFileName = inputFileName.replace(/\.xd$/, `.${awsOutputDir}`);
+let filePath = path.join(tmpDir.name, inputFileName);
+if (conversionType === 'F2S') {
+  outputFileName = `${inputFileName}.sketch`;
```
Comment (kgoedecke) [inline index.js]: Cant you use `awsOutputDir` here as in L13?

**Review by kgoedecke — COMMENTED** [review]
General feedback: PR title should be `feat` not `feature`

File: index.js [hunk 2]
```diff
@@ -9,17 +9,34 @@ const tmpDir = require('./tmp-dir');
 
 const inputFileName = core.getInput('file-name');
 const awsOutputDir = core.getInput('aws-output-directory');
-const outputFileName = inputFileName.replace(/\.xd$/, `.${awsOutputDir}`);
+const conversionType = core.getInput('conversion-type');
+let outputFileName = inputFileName.replace(/\.xd$/, `.${awsOutputDir}`);
```
Comment (kgoedecke) [inline index.js]: So you replace `.xd` with `.sketch` here and then in line 16 you add an other `.sketch` to it? I dont think thats correct, can you double check?

File: index.js [hunk 3]
```diff
@@ -9,17 +9,34 @@ const tmpDir = require('./tmp-dir');
...
+if (conversionType === 'F2S') {
+  outputFileName = `${inputFileName}.sketch`;
+  filePath = inputFileName;
+}
 const awsFileName = `${github.context.issue.number}_${outputFileName}`;
 const messageEnabled = core.getInput('post-message-enabled') === '1';
+const executable = core.getInput('executable');
 
-aws.getFile()
+if (conversionType === 'F2S') {
```
Comment (kgoedecke) [inline index.js:30]: Can we somehow avoid having a if else with almost identical `then` statements.

File: index.js (hunk 2 above)
Comment (ParthJadhav) [reply index.js]: Done !

File: index.js (hunk 1 above)
Comment (ParthJadhav) [reply index.js]: Done !

**Pull request merged**